include/seqan3/alphabet/composition/union_composition.hpp

```diff
@@ -84,11 +87,23 @@ namespace seqan3
  */
 template <typename ...alternative_types>
 //!\cond
-    requires (detail::constexpr_alphabet_concept<alternative_types> && ... && (sizeof...(alternative_types) >= 2))
+    requires (detail::constexpr_alphabet_concept<alternative_types> && ...) &&
+             (sizeof...(alternative_types) >= 2)
+             //TODO same char_type
```

Comment:
TODO

Reply:
can you reference it in the comment then? I think this will always make synchronization easier